scripts/release-ckeditor5.sh

```diff
@@ -10,6 +10,9 @@ echo ""
 
 if [[ $REPLY =~ ^[Yy]$ ]]
 then
+	# Update version of CKEditor 5 dependencies in package.json
+	node ./scripts/release/update-dependencies-versions.js
```

Comment:
It's not part of this script. We need to update versions in `package.json`, commit it, push and wait for CI. It's a separate step.

Reply:
It can. As:

```
git commit -a -m "Internal: Updated dependencies."
```

Reply:
Done.